rmw_zenoh_cpp/src/detail/graph_cache.cpp

```diff
@@ -44,8 +45,17 @@ static std::string generate_base_token(
   const std::string & name)
 {
   std::stringstream token_ss;
-  // TODO(Yadunund): Empty namespace will contain /. Fix non-empty namespace.
-  token_ss << "@ros2_lv/" << domain_id << "/" << entity << namespace_ << name;
+  token_ss << "@ros2_lv/" << domain_id << "/" << entity << namespace_;
+  // An empty namespace from rcl will contain "/"" but zenoh does not allow keys with "//".
+  // Hence we add an "_" to denote an empty namespace such that splitting the key
+  // will always result in 5 parts.
+  if (namespace_ == "/") {
+    token_ss << "_/";
+  } else {
+    token_ss << "/";
+  }
```

Comment:
nit: it can be repaced by one line:

```suggestion
  token_ss << (namespace_ == "/") ? "_/" : "/";
```

Reply:
nit: I think that a docstring for the method with the expected token format will be neat

Reply:
While this could be a one-liner, I actually think it is harder to read that way.  So I left it as-is.